functions/controllers/expressExample/index.js

```diff
@@ -25,8 +31,35 @@ app.get("/", async (req, res) => {
   }
 });
 
+app.get("/resend", jwtCheck, async (req, res) => {
+  try {
+    res.json(req.user.sub);
+  } catch (error) {
+    res.status(500).send({ code: 500, message: "Bad/No key" });
+  }
+});
+
 app.post("/resend", jwtCheck, async (req, res) => {
-  res.status(201).send({ message: "private body reached" });
+  // this endpoint takes in a user_id then converts
+  // it into an API call
+  try {
+    const options = {
+      // options for the API call
+      method: "POST",
+      url: "https://cruzhacks.us.auth0.com/api/v2/jobs/verification-email",
```

Comment:
It's important that every endpoint URL is hidden and that our Auth0 domain is kept a secret to reduce the chances of anyone figuring out what service we are using. The domain listed needs to be kept in an environment variable and string formatted. We do have a different domain from development and production so that does need to be scaled.